src/mlpack/tests/convolution_test.cpp

```diff
@@ -46,13 +45,13 @@ void Convolution2DMethodTest(const arma::mat input,
   // Check the output dimension.
   bool b = (convOutput.n_rows == output.n_rows) &&
       (convOutput.n_cols == output.n_cols);
-  BOOST_REQUIRE_EQUAL(b, 1);
+  REQUIRE(b == 1);
 
   const double* outputPtr = output.memptr();
   const double* convOutputPtr = convOutput.memptr();
 
   for (size_t i = 0; i < output.n_elem; ++i, outputPtr++, convOutputPtr++)
-    BOOST_REQUIRE_CLOSE(*outputPtr, *convOutputPtr, 1e-3);
+    REQUIRE(*outputPtr == Approx(*convOutputPtr).epsilon(1e-3));
```

Comment:
You need to divide by 100, see https://github.com/mlpack/mlpack/pull/2525#discussion_r456959546

Reply:
Ohh, Thanks I'll Correct it.